crates/meta/src/storage.rs

```diff
@@ -8,13 +8,23 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+use codederror::CodedError;
 use restate_schema_impl::SchemasUpdateCommand;
 use serde::{Deserialize, Serialize};
 use std::future::Future;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 use tokio::io;
 use tracing::trace;
 
+type StorageFormatVersionType = u32;
+
+/// Storage format version used by the [`FileMetaStorage`] to store schema information. This value
+/// must be incremented whenever you introduce a breaking change to the schema information.
+const STORAGE_FORMAT_VERSION: StorageFormatVersionType = 0;
```

Comment:
Same as before, let's avoid using version 0 since 0 is the default value of the type.

Reply:
True. Will change it.